log_outgoing_requests/static/log_outgoing_requests/css/admin.css

```
@@ -0,0 +1,6 @@
+.field-res_body {
+	display: inline-block;
+	width: 40rem;
```

Let's make this configurable so we easily tweak in the project where it's used:

```suggestion
	width: var(--dlor-body-width, 40rem);
```